daily-builds.json

```diff
@@ -1,14 +1,3 @@
 {
-  "versions": [
-    "2.0.0",
-    "1.9.0",
-    "1.8.0",
-    "1.8.1",
-    "1.8.2",
-    "1.7.0",
-    "1.7.1",
-    "1.7.2",
-    "1.7.3"
-  ],
   "platforms": "linux/arm64,linux/amd64"
```

Comment:
Why not add this to the workflow matrix?

Reply:
I don't understand, can you elaborate?
We no longer need the static list of supported versions, we are calculating it at runtime now.

Reply:
you're calculating the versions but the json file already exists to hold the list of platforms. I think you can move it to the workflow matrix

Reply:
You mean move the `platforms` list to the workflow Ci so we can get rid of this daily-builds.json file altogether?

Reply:
Just did that change